frontend/src/App.js

```diff
@@ -16,6 +18,8 @@ const App = () => {
         <Route path="/signup" element={!orgUser ? <Signup /> : <Navigate to="/listings" />} />
         <Route path="/login" element={!orgUser ? <Login /> : <Navigate to="/listings" /> } />
         <Route path="/listings" element={<Listings />} />
+        <Route path="/profile" element={<Profile />} />
+        <Route path="/organisations/:org_user_id" element={<OrgProfile />} />
```

Comment:
`<Route path="/organisation/:orgUser_id" element={<OrgProfile />} />`

If that's ok? to keep it consistent with backend

Reply:
we were thinking organisations as this will be the link to all of the registered organisation, eventually there could be a static page totalling all our registered orgs on `/organisations` was my thinking

Reply:
Ah I see, like a portfolio of all the organisations?

Reply:
yeah again im just hinking way ahead just for consistency

i was just looking at other websites and this is the way they do things: 
https://www.gov.uk/government/organisations
https://www.gov.uk/government/organisations/prime-ministers-office-10-downing-street